src/app/app-main/app-main.tsx

```diff
@@ -48,7 +51,9 @@ const AppMain: React.FC = () => {
     <UserContext.Provider value={user}>
       <UserLogInContext.Provider value={logUserIn}>
         <UserLogOutContext.Provider value={logUserOut}>
-          <BrowserRouter>{app}</BrowserRouter>
+          <BrowserRouter>
+            <WorkPackageDetails workPackage={exampleWorkPackage1} />
```

Comment:
Placeholder? It might be easier to insert the component you're working on into `app-authenticated` as an additional page so you don't disrupt the external flow, but it's up to you